### PR TITLE
dotCMS/core#22488 Transform WYSIWYG field to block editor fields

### LINKED ISSUE #22488: Transform WYSIWYG field to block editor fields
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
1. Is content type level
2. We need to update the content in all the contentlets in that content types
3. We might need to wipe the history (so we need a warning)
4. Microservice or client?
5. Enterprise functionality
6. There is no "reverse", once is done is done.
7. No custom blocks supports

**Acceptance criteria**
- Allow users to change the field type
- Support basic blocks: p, h1-h6, marks like bold, link, italic etc.
- Partial support to images, images that are inline inside a `<p>` which is the natural code of the WYSIWYG

### PR DESCRIPTION
**Pull request by fmontes**
### Proposed Changes
* Set HTML ot JSON to the block editor in `edit_field.jsp`
* Fix the `dotcms-block-editor` storybook not starting

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/dot-convert-wysiwyg-to-block/dot-convert-wysiwyg-to-block.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<h3 id="convert" data-testId="infoHeader">
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/dot-convert-wysiwyg-to-block/dot-convert-wysiwyg-to-block.component.html:1]: double IDs `convert`

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp [hunk 2]
```diff
@@ -187,18 +187,27 @@
...
+                let json;
+                let html;
+
+                try {
+                    json = JSON.parse(data);
+                } catch (error) {
+                    html = data;
+                }
+
+                console.log({json, html})
```
Comment (hmoreras) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:203]: remove

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp [hunk 3]
```diff
@@ -187,18 +187,27 @@
...
-                 * it will throw a syntax error
+                 * "textValue" can be a JSON or HTML now
+                 *  In this code we'll indentify if is JSON or HTML and act accordinhly
                  */
-                const data = <%=textValue%>;
+                const data = `<%=textValue%>`
+                let json;
+                let html;
+
+                try {
```
Comment (hmoreras) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:197]: add doc of why the try/cath

**Review by hmoreras — CHANGES_REQUESTED** [review]

**Review by hmoreras — APPROVED** [review]

**Review by alfredo-dotcms — APPROVED** [review]

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp (hunk 3 above)
Comment (jcastro-dotcms) [reply dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:201]: Done!

**Review by alfredo-dotcms — APPROVED** [review]

**Review by nollymar — APPROVED** [review]

**Pull request merged**